app/views/school/home/show_la_funded_place.html.erb

```diff
@@ -0,0 +1,60 @@
+<%- title = t('page_titles.iss.home') %>
+<%- content_for :before_content do %>
+  <%- school_breadcrumbs(school: @school, user: impersonated_or_current_user, items: 'Your account') %>
+<%- end %>
+<%- content_for :title, title %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop govuk-grid-column-full">
+    <span class="govuk-caption-xl">State-funded pupils in independent schools and  alternative provision</span>
+    <h1 class="govuk-heading-xl">
+      <%= title %>
+    </h1>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop govuk-grid-column-full">
+    <h2 class="govuk-heading-l govuk-!-font-size-27">
+      <%= govuk_link_to t('page_titles.iss.order_devices'), order_devices_school_path(@school) %>
+    </h2>
+
+    <p class="govuk-body">Use this section to:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>see how many laptops you have been allocated</li>
```

Comment:
see how many laptops you've been allocated

Reply:
Done